recipe/meta.yaml

```diff
@@ -6,24 +6,24 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flopy-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/flopy-{{ version }}.tar.gz
   sha256: 0ce2941f4095df2ca1d510f28df57224bdeb90636a3f3beeb199f09f635ddc62
 
 build:
   entry_points:
     - get-modflow = flopy.utils.get_modflow:cli_main
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```

Comment:
Do you want to update the build number to 1?

Reply:
this was intentionally not updated, since these changes don't require the package to be re-built and re-distributed for updates.